waspc/data/Generator/templates/sdk/wasp/client/router/linkHelpers.ts

```diff
@@ -17,6 +17,9 @@ export function interpolatePath(
 
 function interpolatePathParams(path: string, params: Params) {
   function mapPathPart(part: string) {
+    if (part === '*') {
```

Comment:
React Router has wild card routes that are called splats, so we need to support users specifying what the `*` part should be. We called that param `splat` to keep in line with the React Router naming.

Reply:
Good catch, `splat` is their secondary name 👍